aioca/_catools.py

```diff
@@ -1081,6 +1096,35 @@ def get_channel(pv: str) -> Channel:
     return channel
 
 
+@dataclass
+class ChannelInfo:
+    """Information about a particular Channel
+
+    Attributes:
+        name: Process Variable name the Channel is targeting
+        connected: True if the Channel is currently connected
+        subscriber_count: Number of clients subscribed to this Channel"""
+
+    name: str
+    connected: bool
+    subscriber_count: int
+
+
+def get_channel_infos() -> List[ChannelInfo]:
+    """Return information about all Channels"""
+    infos: List[ChannelInfo] = []
+    channel_cache = _Context.get_channel_cache()
+
+    for channel in channel_cache.get_channels():
+        infos.append(
+            ChannelInfo(
+                channel.name, channel.connected(), channel.count_subscriptions()
+            )
+        )
+
+    return infos
```

Comment:
Seems a long winded way to write:
```python
return [ChannelInfo(
    channel.name, channel.connected(), channel.count_subscriptions())
    for channel in _Context.get_channel_cache()]
```

Reply:
Implemented in https://github.com/dls-controls/aioca/pull/38/commits/c0fd7c5a491e74344dda518bbd132bf87bd5d9d3